r2/r2/lib/validator/validator.py

```diff
@@ -2736,7 +2736,7 @@ def __init__(self, item, *a, **kw):
 
     def _lookup(self, name, allow_deleted):
         try:
-            return Account._by_name(name, allow_deleted=allow_deleted)
+            return Account._by_name(name.strip(), allow_deleted=allow_deleted)
```

Comment:
> some consistency

I made the change in the effort to keep as much consistency as possible-- in most places it's not stripped and I assume that's a conscious decision. Stripping every time, especially on various get requests where there's a lookup based on previous context (ex cookies), stripping would be pointless. It's only necessary like this on user entry.